src/Web/WebResourceRegistrar.php

```diff
@@ -161,19 +318,36 @@ private function asArray($value): array
         return null;
     }
 
-    private function asMap($value): array
+    /**
+     * Flip given parameter if it array.
+     * Converts to array and then flip parameter if it string.
+     *
+     * @param string|array $value Parameter to flip
+     * @return array|null
+     */
+    private function asMap($value): ?array
     {
         $arr = is_array($value) ? $value : $this->asArray($value);
-        return $arr == null ? null: array_flip($arr);
+
+        return $arr ? array_flip($arr) : null;
     }
 
-    private function removeNamespace(string $controllerClass)
+    /**
+     * Removes namespace from given class.
+     *
+     * @param string $controllerClass Class to remove namespace
+     *
+     * @return string
+     */
+    private function removeNamespace(string $controllerClass): string
```

Comment:
Isn't it the same as **getShortClassName**?

Reply:
If i right understand logic of this method, here return short class name if it class and given parameter if it not. In this way it's not a getSHortClassName and also getSHortClassName already exists in this class. So maybe something like: "prepareControllerName"